lms/templates/courseware/course_about.html

```diff
@@ -18,6 +18,25 @@
   ## (https://developers.facebook.com/docs/opengraph/howtos/maximizing-distribution-media-content#tags)
   <meta property="og:title" content="${course.display_name_with_default_escaped}" />
   <meta property="og:description" content="${get_course_about_section(request, course, 'short_description')}" />
+
+  <!-- Google Code for Lead Conversion Page -->
+  <script type="text/javascript">
+  /* <![CDATA[ */
+  var google_conversion_id = 838419445;
+  var google_conversion_language = "en";
+  var google_conversion_format = "3";
+  var google_conversion_color = "ffffff";
+  var google_conversion_label = "0WzCCOuDrnUQ9YfljwM";
+  var google_remarketing_only = false;
+  /* ]]> */
+  </script>
+  <script type="text/javascript" src="//www.googleadservices.com/pagead/conversion.js">
+  </script>
+  <noscript>
+  <div style="display:inline;">
+  <img height="1" width="1" style="border-style:none;" alt="" src="//www.googleadservices.com/pagead/conversion/838419445/?label=0WzCCOuDrnUQ9YfljwM&amp;guid=ON&amp;script=0"/>
+  </div>
+  </noscript>
```

Comment:
nit: this code matches the [addition below in contact.html](https://github.com/open-craft/edx-theme/pull/19/files#diff-a7704ef1d848aac7e331ddcb5c96080cR7) -- could be pulled into a separate file.